griptape/tools/web_scraper/tool.py

```diff
@@ -24,7 +23,7 @@ class WebScraper(BaseTool):
     })
     def get_content(self, params: dict) -> ListArtifact | ErrorArtifact:
         url = params["values"]["url"]
-        page = self._load_page(url)
+        page = WebLoader()._load_page(url, self.include_links)
```

Comment:
@vasinov Not sure if we should be using this protected method like this.

Reply:
Let's just make it public.